R/info.R

```diff
@@ -0,0 +1,70 @@
+##' Return info about a report which has been run
+##'
+##' This will return info from either successful or failed reports. It will
+##' look for the report with `id` in archive first and then look in drafts
+##' if it can't be found from archive.
+##'
+##' @param id The report ID
+##' @param name The name of the report
+##' @inheritParams orderly_list
+##'
+##' @return Info from report run
```

Comment:
Do we want to advertise the exact format is subject to change? If not do you want to document it?

Reply:
Thanks, added a note saying it is subject to change. And added some details too about what it returns at the moment.